### ISSUE TITLE
[feature request] a hotkey for immediately closing a pinned tab

### ISSUE THREAD

**Issue by 07416** [jayesh-bhoot/pin-unpin-tab#18]
Not natively supported by Firefox to my knowledge -- pinned tabs must be first "unpinned". Would complement the existing shortcut:

![image](https://user-images.githubusercontent.com/14923168/115232789-ccfb2000-a11f-11eb-9656-a31dda622f4a.png)

**Comment by jayesh-bhoot** [comment]
Yeah this has been floating in my mind since a few days. It’s worth implementing.

**Comment by 07416** [comment]
Alt+W or Alt+X hotkey by default?
